wire-library/wire-runtime-swift/src/main/swift/ProtoEnum.swift

```diff
@@ -64,7 +64,7 @@ extension ProtoEnum {
 }
 
 /// This type gives access to `decoder.enumDecodingStrategy`
-private struct BoxedEnum<T: ProtoEnum> : Decodable {
+struct BoxedEnum<T: ProtoEnum> : Decodable {
```

Comment:
Can this be `internal`?

Reply:
It's implicitly internal

Reply:
Oh I swear it said `public` when I looked at it. Maybe that was before the rebase.